opal/runtime/opal_progress.h

```diff
@@ -142,7 +142,7 @@ OPAL_DECLSPEC void opal_progress_set_event_poll_rate(int microseconds);
  *
  * Prototype for the a progress function callback.  Progress function
  * callbacks can be registered with opal_progress_register() and
- * deregistered with opal_progress_deregister().  It should be noted
+ * deregistered with opal_progress_unregister().  It should be noted
```

Comment:
Please change back this one too.

Reply:
This one fixes the doxygen doc to reflect the effectively used `opal_progress_unregister()` function (line 170). 
Or may you want to rename the function everywhere back to `opal_progress_deregister()` ?

Reply:
all good, my bad.